aws/client.go

```diff
@@ -92,6 +96,13 @@ func (t *ResilientTransport) tries(req *http.Request) (res *http.Response, err e
 	return
 }
 
+func RandExpBackoff(try int) {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	time.Sleep(time.Duration(r.Intn(1000)) *
+		time.Millisecond *
+		time.Duration(math.Exp2(float64(try))))
+}
+
```

Comment:
The idea is that If you have multiple clients trying to call AWS API, they tend to experience throttling at the same time. Then they wait for exactly the same amount of time and retry, which lead to another round of throttling. Repeat for next retries.
Randomness doesn't allow retries to line up.